lib/helpers/view_helper.ex

```diff
@@ -46,7 +48,15 @@ defmodule Aprb.ViewHelper do
   end
 
   def exchange_admin_link(order_id) do
-    "https://exchange.artsy.net/admin/orders/#{order_id}"
+    "#{@exchange_url}/admin/orders/#{order_id}"
+  end
+
+  def exchange_partner_orders_link(partner_id) do
+    "#{@exchange_url}/admin/orders?q%5Bseller_id_eq=#{partner_id}"
+  end
+
+  def exchange_user_orders_link(user_id) do
+    "#{@exchange_url}/admin/orders?q%5Bbuyer_id_eq=#{user_id}"
```

Comment:
the `buyer_id_eq` is sort of an odd format, is that what Exchange accepts?

Reply:
yes, its `activeadmin` magic :)

Reply:
Weirddd.